laplace/baselaplace.py

```diff
@@ -961,6 +1020,18 @@ def prior_precision(self, prior_precision):
         if len(self.prior_precision) not in [1, self.n_layers]:
             raise ValueError('Prior precision for Kron either scalar or per-layer.')
 
+    def state_dict(self) -> dict:
+        state_dict = super().state_dict()
+        state_dict['H'] = self.H_facs.kfacs
+        return state_dict
+
+    def load_state_dict(self, state_dict: dict):
+        super().load_state_dict(state_dict)
+        self._init_H()
```

Comment:
I think this will break the `override=False` use case. Maybe add a test for this case.